tests/algorithms/test_bc.py

```diff
@@ -94,15 +94,14 @@ def test_train_end_cond_error(trainer: bc.BC):
 
 
 def test_bc(trainer: bc.BC, cartpole_venv):
-    sample_until = rollout.make_min_episodes(15)
-    novice_ret_mean = rollout.mean_return(trainer.policy, cartpole_venv, sample_until)
+    novice_ret_mean, _ = evaluation.evaluate_policy(trainer.policy, cartpole_venv, 15)
+
     trainer.train(
         n_epochs=1,
         on_epoch_end=lambda: print("epoch end"),
         on_batch_end=lambda: print("batch end"),
     )
-    trainer.train(n_batches=10)
```

Comment:
why is this line being deleted?

Reply:
That was an accident. I will add it back in.